app/renderer/js/electron-bridge.ts

```diff
@@ -1,17 +1,17 @@
-const events = require('events');
-const { ipcRenderer } = require('electron');
+import events from 'events';
+import { ipcRenderer } from 'electron';
 
 // we have and will have some non camelcase stuff
 // while working with zulip so just turning the rule off
 // for the whole file.
-/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/camelcase */
 class ElectronBridge extends events {
-	send_event(...args) {
-		this.emit(...args);
+	send_event(eventName: string | symbol, ...args: any[]): void {
+		this.emit(eventName, ...args);
 	}
 
-	on_event(...args) {
-		this.on(...args);
+	on_event(eventName: string, listener: ((...args: any[]) => void)): void {
```

Comment:
Minor suggestion here, let move the type into a variable for listener and use it here, it will look more cleaner that way.